src/doc/book/closures.md

```diff
@@ -492,12 +492,12 @@ fn factory() -> Box<Fn(i32) -> i32> {
 
     Box::new(move |x| x + num)
 }
-# fn main() {
+fn main() {
 let f = factory();
 
 let answer = f(1);
 assert_eq!(6, answer);
-# }
+}
```

Comment:
No worries, it's a rust-specific thing. We wouldn't want to, for example, run the full test suite for a tiny change like this, so we "roll up" a bunch of them together and run the tests for them all at once.

Adding the spaces would be a totally new pull request.


On Apr 15, 2016, 18:08 -0400, Kaiyin Zhong<notifications@github.com>, wrote:
>  
> Insrc/doc/book/closures.md(https://github.com/rust-lang/rust/pull/32973#discussion_r59948089):
>  
> >let f = factory();>>let answer = f(1);>assert_eq!(6, answer);>-# }>+}
>  
> To be honest I am a bit confused by the rollups below. Not yet very good at git. 😄
>  
>  
> —
> You are receiving this because you were mentioned.
> Reply to this email directly orview it on GitHub(https://github.com/rust-lang/rust/pull/32973/files/f91649144e1e2e6b3664545cb7e117d839f9cd95#r59948089)
>  
>  
>